### PR TITLE
f-content-cards@1.6.0: Add emitters

### PR DESCRIPTION
**Pull request by asherstoppard**
Add the following event emitters so the client can hook into certain aspects of f-metadata through the f-content-cards component:

- `@on-appboy-init`
- `@get-card-count`
- `@has-loaded`
- `@on-error`

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

### CODE REVIEW

**Review by DamianMullins — DISMISSED** [review]

File: packages/f-content-cards/src/components/ContentCards.vue [hunk 1]
```diff
@@ -178,9 +182,14 @@ export default {
                 .getTitleCard()
                 .arrangeCardsByTitles()
                 .output();
+
             this.cards = cards;
             this.titleCard = titleCard;
             this.hasLoaded = true;
+
+            this.$emit('on-appboy-init', appboy);
+            this.$emit('get-card-count', cards.length);
```
Comment (kevinrodrigues) [inline packages/f-content-cards/src/components/ContentCards.vue:191]: Would card return as an empty array if it doesn't come back with anything?

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/f-content-cards/src/components/ContentCards.vue [hunk 2]
```diff
@@ -178,9 +182,14 @@ export default {
                 .getTitleCard()
                 .arrangeCardsByTitles()
                 .output();
+
             this.cards = cards;
             this.titleCard = titleCard;
             this.hasLoaded = true;
+
+            this.$emit('on-appboy-init', appboy);
```
Comment (minrwhite) [inline packages/f-content-cards/src/components/ContentCards.vue]: 1. appboy is an old name for braze - would `on-braze-init` be better in an external (to the package) interface?

Comment (asherstoppard) [reply packages/f-content-cards/src/components/ContentCards.vue]: That's a good point! I'll get it changed over :+1:

File: packages/f-content-cards/src/components/ContentCards.vue (hunk 1 above)
Comment (asherstoppard) [reply packages/f-content-cards/src/components/ContentCards.vue:191]: That's correct, it defaults to an empty array in the content card constructor :+1:

**Review by DamianMullins — APPROVED** [review]

**Review by minrwhite — APPROVED** [review]

**Pull request merged**